### PR TITLE
Fixing message name that should not take a capital first letter automatically

### PR DESCRIPTION
**Pull request by flonou**
This fixes a small issue where message name variable was capitalized when it shouldn't (making ros unable to recognize the message type)

### CODE REVIEW

**Comment by LaurieCheers-unity** [comment]
Interesting. This logic was unchanged from ROS#: https://github.com/siemens/ros-sharp/blob/327c60be19bc6953104fb8c6df88d003f30f1dbb/Libraries/MessageGeneration/MessageParser.cs#L70 but I guess the difference is they don't use it for message importing...?

**Comment by LaurieCheers-unity** [comment]
Could you provide more context about why this needed to change? ROS official guidance is that all message names should start with a capital letter: http://wiki.ros.org/ROS/Patterns/Conventions#line-39 so it feels like that's the "correct" fix.

**Comment by flonou** [comment]
Indeed, they do say to use a capital first letter.
I just encountered the issue today using messages someone else wrote and ros was importing the message type with a lowercase on wsl2 side (because the filename first letter was lowercase) using catkin_make.
I guess lowercase first letter is bad practice but ros seems to not fail when encountering it so the best options might be :
- Display a warning when generating and capitalize
- Display a warning when generating and don't capitalize
- Don't capitalize

File: Runtime/MessageGeneration/MessageParser.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ public MessageParser(List<MessageToken> tokens, string outPath, string rosPackag
 
             if (rosMsgName.Equals(""))
             {
-                this.rosMsgName = MsgAutoGenUtilities.CapitalizeFirstLetter(inFileName);
+                this.rosMsgName = inFileName;
```
Comment (mrpropellers) [inline Runtime/MessageGeneration/MessageParser.cs:75]: Could you add a Debug.LogWarning here to warn the user that the message name was not found and is being inferred from the filename?  That could help prevent some confusion in the future around what's happening here.

**Review by mrpropellers — CHANGES_REQUESTED** [review]

File: Runtime/MessageGeneration/MessageParser.cs (hunk 1 above)
Comment (flonou) [reply Runtime/MessageGeneration/MessageParser.cs]: I added a warning message, is that what you meant ?

**Review by mrpropellers — APPROVED** [review]

File: Runtime/MessageGeneration/MessageParser.cs (hunk 1 above)
Comment (mrpropellers) [reply Runtime/MessageGeneration/MessageParser.cs]: Yep looks good!

**Review by LaurieCheers-unity — CHANGES_REQUESTED** [review]
Should be merged into dev.

**Review by mrpropellers — APPROVED** [review]

**Review by LaurieCheers-unity — APPROVED** [review]

**Pull request merged**